pkg/config/config.go

```diff
@@ -1266,13 +1266,16 @@ func (c *Config) RemovedVariableCheck(confFile string) error {
 // Load loads config options from a toml file.
 func (c *Config) Load(confFile string) error {
 	metaData, err := toml.DecodeFile(confFile, c)
+	if err != nil {
+		return err
```

Comment:
Does this err will display in log file? It user set a wrong config, and it's better to let they konw why tidb cannot start.

Reply:
Yes, the error log will like `toml: line 5 (last key \"performance.enforce-mpp\"): incompatible types: TOML value has type int64; destination has type boolean`

Reply:
<img width="1919" alt="image" src="https://github.com/pingcap/tidb/assets/17435596/282a81e6-5e0f-4f11-a6f6-dd73228738f9">

Reply:
I'm curious why it works because I remeber `global config` is inited before `logger`. If `global config` init failed, `logger` will not work

Reply:
Use `log.Fatal`. 😂